boards/stm32f723e-disco/include/board.h

```diff
@@ -26,6 +26,101 @@
 extern "C" {
 #endif
 
+/**
+ * @name    LCD Backlight control defines
+ * @{
+ */
+#define BACKLIGHT_PIN       GPIO_PIN(PORT_H, 11)    /**< Backlight pin (HIGH active LD_BL) */
+#define BACKLIGHT_MASK      (1 << 11)               /**< Backlight pin mask */
+
+/** Set the backlight pin */
+#define BACKLIGHT_ON        (GPIOH->BSRR = BACKLIGHT_MASK)
+/** Clear the backlight pin */
+#define BACKLIGHT_OFF       (GPIOH->BSRR = (BACKLIGHT_MASK << 16))
+/** Toggle the backlight pin */
+#define BACKLIGHT_TOGGLE    (GPIOH->ODR  ^= BACKLIGHT_MASK)
+/** @} */
+
+/**
+ * @name    LCD display definitions
+ *
+ * The STM32F723G-DISCO board has a 240 x 240 pixel TFT color LCD display with
+ * ST7789H2 driver IC using the MCU 8080 16-bit parallel interface (default with
+ * SB15=ON and SB16=OFF). SB15 and SB16 control the ST7789H2 `IM0` signal.
+ * With SB15=OFF and SB16=ON, the MCU 8080 8-bit parallel interface is enabled.
+ *
+ * For the `LCD_*` pins the identifiers are used as given in the [schematic]
+ * (https://www.st.com/resource/en/schematic_pack/mb1260-f723e-d03_schematic.pdf).
+ *
+ * The LCD display has to switched on explicitly by activating VDD_LCD using
```

Comment:
```suggestion
 * The LCD display has to be switched on explicitly by activating VDD_LCD using
```

Reply:
:+1: Good catch, this was copied & pasted from the `stm32l496g-disco` board, which is not applicable to the `stm32f723e-disco` board at all. I removed it completely and added a commit for the `stm32l496g-disco` board to fix the doc.